Code/Tools/ProjectManager/Source/GemRepo/GemRepoScreen.cpp

```diff
@@ -287,46 +292,46 @@ namespace O3DE::ProjectManager
 
         connect(addRepoButton, &QPushButton::clicked, this, &GemRepoScreen::HandleAddRepoButton);
 
-        topMiddleHLayout->addSpacing(30);
-
         middleVLayout->addLayout(topMiddleHLayout);
 
         middleVLayout->addSpacing(30);
 
-        // Create a QTableWidget just for its header
-        // Using a seperate model allows the setup  of a header exactly as needed
-        m_gemRepoHeaderTable = new QTableWidget(this);
-        m_gemRepoHeaderTable->setObjectName("gemRepoHeaderTable");
-        m_gemRepoListHeader = m_gemRepoHeaderTable->horizontalHeader();
-        m_gemRepoListHeader->setObjectName("gemRepoListHeader");
-        m_gemRepoListHeader->setDefaultAlignment(Qt::AlignLeft);
-        m_gemRepoListHeader->setSectionResizeMode(QHeaderView::ResizeMode::Fixed);
-
-        // Insert columns so the header labels will show up
-        m_gemRepoHeaderTable->insertColumn(0);
-        m_gemRepoHeaderTable->insertColumn(1);
-        m_gemRepoHeaderTable->insertColumn(2);
-        m_gemRepoHeaderTable->setHorizontalHeaderLabels({ tr("Repository Name"), tr("Creator"), tr("Updated") });
-
-        const int headerExtraMargin = 18;
-        m_gemRepoListHeader->resizeSection(0, GemRepoItemDelegate::s_nameMaxWidth + GemRepoItemDelegate::s_contentSpacing + headerExtraMargin);
-        m_gemRepoListHeader->resizeSection(1, GemRepoItemDelegate::s_creatorMaxWidth + GemRepoItemDelegate::s_contentSpacing);
-        m_gemRepoListHeader->resizeSection(2, GemRepoItemDelegate::s_updatedMaxWidth + GemRepoItemDelegate::s_contentSpacing);
-
-        // Required to set stylesheet in code as it will not be respected if set in qss
-        m_gemRepoHeaderTable->horizontalHeader()->setStyleSheet("QHeaderView::section { background-color:transparent; color:white; font-size:12px; border-style:none; }");
+        constexpr int headerTableMinWidth = MinWindowWidth - inspectorWidth - middleLayoutIndent * 2;
```

Comment:
Is headerTableMinWidth actually used in this function?

Reply:
nope